docs/topics/cli-args.adoc

```diff
@@ -38,7 +38,15 @@ This option may result in a longer execution time and a large number of migratio
 |--enableClassNotFoundAnalysis |Flag to enable analysis of Java files that are not available on the class path. This should not be used if some classes will be unavailable at analysis time.
 |--enableCompatibleFilesReport |Flag to enable generation of the Compatible Files report. Due to processing all files without found issues, this report may take a long time for large applications.
 |--enableTattletale |Flag to enable generation of a Tattletale report for each application. This option is enabled by default when `eap` is in the included target. If both `enableTattletale` and `disableTattletale` are set to true, then `disableTattletale` will be ignored and the Tattletale report will still be generated.
-|--excludePackages |A space-delimited list of packages to exclude from evaluation. For example, entering `com.mycompany.commonutilities` would exclude all classes whose package name begins with `com.mycompany.commonutilities`.
+|--enableTransactionAnalysis |[Technology Preview] Flag to enable generation of a Transactions Report that displays the call stack, which executes operations on relational database tables. It supports Spring Data JPA and the traditional preparedStatement() method for SQL statement execution. It does not support ORM frameworks, such as Hibernate.
```

Comment:
"It supports Spring Data" -- it would be clearer if you used the noun or noun phrase you have in mind here. I think that would be "The Enable Transaction action analysis feature" . The "it" in the last sentence is fine; no need to replace it.  
preparedStatement() -- needs backticks before and after. Basic rule of thumb: anything that you'd write using Courier in Word needs `backticks` in asciidoc.